stats.go

```diff
@@ -14,8 +14,13 @@ import (
 	"go.opencensus.io/stats/view"
 )
 
-// defaultStatsAddr specifies the default address for the DogStatsD service.
-const defaultStatsAddr = "localhost:8125"
+const (
+	// DefaultStatsAddrUDP specifies the default protocol (UDP) and address for the DogStatsD service.
+	DefaultStatsAddrUDP = "localhost:8125"
+
+	// DefaultStatsAddrUDS specifies the default socket address for the DogStatsD service over UDS.
+	DefaultStatsAddrUDS = "unix:///var/run/datadog/dsd.socket"
```

Comment:
What happens on systems that aren't UNIX? Should we move this constant into a file that builds only for supporting environments using the `+build` directive?